greenplum-db/GreenplumR
# Update GreenplumR test

1. Rename PivotalR to GreenplumR
2. Change database name to "test"
3. Enable R CMD check
4. Remove PivotalR design doc
5. Refactor test script

Co-authored-by: Violet Cheng <vcheng@pivotal.io>
Co-authored-by: Haozhou Wang <hawang@pivotal.io>
